subsys/bluetooth/controller/Kconfig

```diff
@@ -554,6 +554,22 @@ config BT_CTLR_PERIPHERAL_ISO
 	bool "LE Connected Isochronous Stream Peripheral [EXPERIMENTAL]" if BT_LL_SW_SPLIT
 	select BT_CTLR_SET_HOST_FEATURE
 
+config BT_CTLR_CONN_ISO_GROUPS
+	int "LE Connected Isochronous Groups"
+	depends on BT_CTLR_CENTRAL_ISO || BT_CTLR_PERIPHERAL_ISO
+	range 1 240
+	default 2
+	help
+	  Maximum supported CIGs.
+
+config BT_CTLR_CONN_ISO_STREAMS
+	int "LE Connected Isochronous Streams per Group"
+	depends on BT_CTLR_CENTRAL_ISO || BT_CTLR_PERIPHERAL_ISO
+	range 1 31
+	default 1
+	help
+	  Maximum supported CISes per CIG.
```

Comment:
I think this is fine for now. I do wonder if a collective pool of streams would be a better solution (that's what we have implemented in the host). 

Imagine a use case where you need typically need to do 2 CIG with 2 CiS each, but once in a while you need to 1 CIG with 5 CIS and occasionally 1 BIG with 5 BIS, but never at the same time. That would be a total of 10 CIS and 5 BIS streams you need to preallocate, even if you never use more than 5 streams (CIS or BIS) at a time. 

Of course you are probably more aware of the potential usecases than I am, but I think there might be some optimization to be found by using a collective pool of streams, possibly even using the same streams for CIS/BIS.

As mentioned this is fine for now, but it's food for thought :)

Reply:
I think a pool of CISes would be a good idea as an optimization. It will not necessarily make this setting go away though. If we have a pool of 4 CISes and but every CIG can contain at most 2 CISes, we only want 2 pointers to CISes in a CIG.

Reply:
On second thought, let's get this right from the beginning and have separate pools for CIGs and CISes.